views/widget.scout.display.php

```diff
@@ -19,20 +19,21 @@
     // Grab the settings from $instance and fill out default values as needed.
 	$title = empty( $instance['title'] ) ? ' ' : apply_filters( 'widget_title', $instance['title'] );
 ?>
-	<script type="text/javascript">
+<?php if ($use_spm): ?>
+    <div class="sailthru-spm-widget">
+         <div id="<?php echo $this->id; ?>"></div>
+          <script type="text/javascript">
+            jQuery(function() {
+                SPM.addSection('<?php echo $section; ?>', {
```

Comment:
If using SPM, will there always be a section? Is this $use_spm set separately from using it at all?

Also, why not rename the file to widget.spm.display.php or widget.section.display?